src/Shader.h

```diff
@@ -20,11 +20,7 @@ class Shader
 
 private:
     static const unsigned int NUM_SHADERS = 2;
-    enum
-    {
-        TRANSFORM_U,
-        NUM_UNIFORMS
-    };
```

Comment:
What was wrong with using this enum? It seems clear to me on line 25 of `Shader.cpp`